src/test/scala/uk/gov/hmrc/ui/pages/ReportElectionsPage.scala

```diff
@@ -20,19 +20,16 @@ object ReportElectionsPage extends BasePage {
 
   override val pageUrl: String = baseUrl + "/elections/report-elections"
 
-  def checkPage(): Unit = {
+  def makeElectionsYes(): Unit = {
     onPage(pageUrl)
+    click(yesRadioId)
+    click(submitButtonId)
+  }
```

Comment:
These methods can be moved to BasePage to reduce duplication, as they perform the same actions with different radio button selections.